src/User32.Desktop/USer32+HELPINFO.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Runtime.InteropServices;
+
+    /// <content>
+    /// Contains the <see cref="HELPINFO"/> nested type.
+    /// </content>
+    public partial class User32
+    {
+        [StructLayout(LayoutKind.Sequential)]
+        public struct HELPINFO
+        {
+            public uint cbSize;
+            public int iContextType;
+            public int iCtrlId;
+            public IntPtr hItemHandle;
+            public IntPtr dwContextId;
```

Comment:
`dwContextId` is typed as a `DWORD`, which doesn't change size with architecture. So this should be `uint` instead.